videohelpersuite/nodes.py

```diff
@@ -600,23 +637,23 @@ def pad(image):
                 output_files.append(output_file_with_audio_path)
                 #Return this file with audio to the webui.
                 #It will be muted unless opened or saved with right click
-                file = output_file_with_audio
+                preview['filename'] = output_file_with_audio
```

Comment:
Yes, this isn't entirely complete because if there is audio, the metadata will still only point to the intermediate video file, but that's kept by default anyway, and this is still better than it was and covers most cases.